code/modules/clothing/outfits/ert.dm

```diff
@@ -466,7 +466,7 @@
 		return
 
 	var/obj/item/radio/headset = H.ears
-	headset.keyslot = new /obj/item/encryptionkey/heads/hos
+	headset.keyslot = new /obj/item/encryptionkey/headset_com
```

Comment:
just give them command radios

Reply:
They're given Centcomm radios so they have access to the centcomm channel in addition to the encryption keys defined here (for each marine type), if i'm reading this correctly. If I just gave them command headsets they wouldn't have their own channel to communicate on.